Extensions/InternetConnectivity.json

```diff
@@ -0,0 +1,32 @@
+{
+  "author": "The Gem Dev",
+  "description": "",
+  "extensionNamespace": "",
+  "fullName": "Internet Connectivity",
+  "name": "InternetConnectivity",
+  "shortDescription": "an extension which checks if the device is connected to the internet. ",
+  "tags": "javascript, desktop, mobile, internet, connection",
```

Comment:
Set: `javascript, internet, network connection, online`

(desktop, mobile is not useful here)

Reply:
okay